core/src/types.rs

```diff
@@ -391,6 +427,7 @@ impl Default for SyncClientConfig {
 			confidence: 99.9,
 			disable_rpc: false,
 			app_id: None,
+			network_mode: NetworkMode::Both,
```

Comment:
We can remove `disable_rpc` entirely and use equality directly in the code, since we are storing network_mode in the configuration.